.circleci/config.yml

```diff
@@ -597,7 +597,22 @@ jobs:
             npm --version
             test/solcjsTests.sh /tmp/workspace/soljson.js $(cat /tmp/workspace/version.txt)
 
-  t_ems_external_gnosis:
+  t_ems_compile_ext_gnosis:
+    docker:
+      - image: circleci/node:10
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: External GnosisSafe compilation
+          command: |
+            export COMPILE_ONLY=1
+            test/externalTests/gnosis.sh /tmp/workspace/soljson.js || test/externalTests/gnosis.sh /tmp/workspace/soljson.js
+
+  t_ems_test_ext_gnosis:
```

Comment:
This one now redoes the compilation part of the new one, right?

Reply:
Yes, the job that's running on every PR is only compiling the contracts, the nightly one does compile + tests.

Reply:
I guess ideally we could reuse the compilation part, like we do in our own build/test jobs. Would that be a lot more work?

Reply:
I'm not sure if I'm getting the question correctly, but this flag only specifies if `npm test` should be run or not. `npm compile` is called in any case. It differs from our compilation tests, in the sense that the ext. ones are run through `solc-js`. Since we just decided to remove testing of external projects completely, I'll update this PR accordingly.